content/doc/features/authentication.en-us.md

```diff
@@ -13,6 +13,180 @@ menu:
+    authenticate. The `%s` matching parameter will be substituted with login
+    name given on sign-in form.
+  - Example: `(&(objectClass=posixAccount)(uid=%s))`
+  - To substitute more than once `%[1]s` should be used instead, eg. when
+    matching supplied login name against multiple attributes such as user
+    identifier, email or even phone number.
+  - Example: `(&(objectClass=Person)(|(uid=%[1]s)(mail=%[1]s)(mobile=%[1]s)))`
+
+**LDAP using simple auth** adds the following fields:
+
+- User DN **(required)**
+  - A template to use as the user's DN. The `%s` matching parameter will be substituted with login name given on sign-in form.
+  - Example: `cn=%s,ou=Users,dc=mydomain,dc=com`
+  - Example: `uid=%s,ou=Users,dc=mydomain,dc=com`
+
+- User Filter **(required)**
+  - An LDAP filter declaring when a user should be allowed to log in. The `%s`
+    matching parameter will be substituted with login name given on sign-in
+    form.
+  - Example: `(&(objectClass=posixAccount)(cn=%s))`
+  - Example: `(&(objectClass=posixAccount)(uid=%s))`
+
+**Verify group membership in LDAP** uses the following fields:
+
+* Group Search Base (optional)
+    * The LDAP DN used for groups.
+    * Example: `ou=group,dc=mydomain,dc=com`
+
+* Group Name Filter (optional)
+    * An LDAP filter declaring how to find valid groups in the above DN.
+    * Example: `(|(cn=gogs_users)(cn=admins))`
+
+* User Attribute in Group (optional)
+    * Which user LDAP attribute is listed in the group.
+    * Example: `uid`
+
+* Group Attribute for User (optional)
+    * Which group LDAP attribute contains an array above user attribute names.
+    * Example: `memberUid`
+
+## PAM
+
+To configure this you just need to set the 'PAM Service Name' to a filename in `/etc/pam.d/`.
+If you want it to work with normal Linux passwords, the user running Gogs must have read access to `/etc/shadow`.
+
+## SMTP
+
+This option allows Gogs to log in to your SMTP host as a Gogs user. To configure this, simply set the fields below:
+
+- Authentication Name **(required)**
+  - A name to assign to the new method of authorization.
+
+- SMTP Authentication Type **(required)**
+  - Type of authentication for use on your SMTP host, PLAIN or LOGIN.
+
+- Host **(required)**
+  - The address where the SMTP host can be reached.
+  - Example: `smtp.mydomain.com`
+
+- Port **(required)**
+  - The port to use when connecting to the server.
+  - Example: `587
+
+- Allowed Domains
+  - Restrict what domains can log in if you're using public SMTP host or SMTP host with multiple domains.
+  - Example: `gogs.io,mydomain.com,mydomain2.com`
+
+- Enable TLS Encryption
+  - Enable TLS encryption on authentication.
+
+- Skip TLS Verify
+  - Disable TLS verify on authentication.
+  
+- This authentication is activate
+  - Enable or disable this auth.
+
+## Freeipa
+
+- In order to login to the Gogs using FreeIPA credentials, you need to create a bind account for Gogs to use:
```

Comment:
changing the doc part to Gitea but, are the config files in this doc still point to gogs or should be changed to gitea too?